tietokanta/src/main/resources/db/migration/V1_1302__.sql

```diff
@@ -0,0 +1,112 @@
+-- ---------------------------------------------------------------------
+-- Tehtävä- ja määrätoteumiin perustuvat tavoitehintamuutokset
+-- ---------------------------------------------------------------------
+
+CREATE TYPE muutos_yksikkohinta_lahde_enum AS ENUM ('laskettu','aseta','manuaali');
```

Comment:
Nyt olisi vielä helpommin mahdollista muokata enumit vähän selkeämpään muotoon, ennen kuin mergetään.

esim. 
1. laskettu -> laskettu (pysyy ennallaan)
2. aseta -> valittu (eli valittu yksikköhinta)
3. manuaali -> syotetty-tavoitehinta (eli, tavoitehinta on syötetty manuaalisesti)

Näin ainakin aseta ja manuaali ei sekoitu merkitykseltään niin helposti toisiinsa ja näkee selkeämmin mikä lähde on kyseessä.